From 8ac2efc71a0175784ea0a27b3a121df915b8c95b Mon Sep 17 00:00:00 2001
Message-Id: <patch-01.31-8ac2efc71a0-20210528T110027Z-avarab@gmail.com>
In-Reply-To: <cover-00.31-00000000000-20210528T110027Z-avarab@gmail.com>
References: <87lf80l1m6.fsf@evledraar.gmail.com>
	<cover-00.31-00000000000-20210528T110027Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 28 May 2021 00:41:56 +0200
Subject: [PATCH 01/31] hooks tests: don't leave "actual" nonexisting on
 failure
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Start by creating an "actual" file in a core.hooksPath test that has
the hook echoing to the "actual" file. We later test_cmp that file to
see what hooks were run. Having that list of files be empty if we fail
instead of throwing an error about it being nonexistent makes for
friendlier output, we'll see what hooks we should have run.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/t1350-config-hooks-path.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/t/t1350-config-hooks-path.sh b/t/t1350-config-hooks-path.sh
index f1f9aee9f5d..fa9647a7c0b 100755
--- a/t/t1350-config-hooks-path.sh
+++ b/t/t1350-config-hooks-path.sh
@@ -5,6 +5,7 @@ test_description='Test the core.hooksPath configuration variable'
 . ./test-lib.sh
 
 test_expect_success 'set up a pre-commit hook in core.hooksPath' '
+	>actual &&
 	mkdir -p .git/custom-hooks .git/hooks &&
 	write_script .git/custom-hooks/pre-commit <<-\EOF &&
 	echo CUSTOM >>actual
-- 
2.32.0.rc1.458.gd885d4f985c

